File: py/hw/shapes2.py
#date: 9/22/2022.
#description:
	#print rectangle, ellipse, arrow, & rhombus using asterisks(*).

#define shapes.
rect:str                =	"*****\n*   *\n*   *\n*   *\n*****"
ellipse:str             =	"  ***  \n *   * \n*     *\n *   * \n  ***  "
arrow:str		=	"  *  \n *** \n*****\n  *  \n  *  "
rhombus:str	        =	"  *  \n * * \n*   *\n * * \n  *  "

#output shapes to console.
print(rect+'\n')
print(ellipse+'\n')
print(arrow+'\n')
print(rhombus)

File: java/hw/Shapes2.java
/*
 * #period: 1.
 * #date: 5/12/2023.
 * #description:
 * 	#print rectangle, ellipse, arrow, & rhombus using asterisks(*).
*/
public class Shapes2{
	public static void main(String[] args){
		String rect		=	"*****\n*   *\n*   *\n*   *\n*****";
		String ellipse	=	"  ***  \n *   * \n*     *\n *   * \n  ***  ";
		String arrow	=	"  *  \n *** \n*****\n  *  \n  *  ";
		String rhombus	=	"  *  \n * * \n*   *\n * * \n  *  ";

		System.out.println(rect+"\n");
		System.out.println(ellipse+"\n");
		System.out.println(arrow+"\n");
		System.out.println(rhombus);
	}
}